### ISSUE TITLE
about show metric

### ISSUE THREAD

**Issue by Kevin-Moon** [tflearn/tflearn#743]
when I train the nets,

It shows the metric for "every step".
it is very useful to know weather the training is okay now.

but it's very hard to find top-10 high validation accuracy epoch among whole epochs.
so I only see the metric result only every epoch without every step. is this possible?

if not, I want to hear other recommendation.

thank you for nice and handy wrapper!

Training Step: 20823  | total loss: 0.48593 | time: 122.170s
| Momentum | epoch: 033 | loss: 0.48593 - acc: 0.8400 -- iter: 16096/20311
Training Step: 20824  | total loss: 0.47759 | time: 122.431s
| Momentum | epoch: 033 | loss: 0.47759 - acc: 0.8435 -- iter: 16128/20311
Training Step: 20825  | total loss: 0.49496 | time: 122.716s
| Momentum | epoch: 033 | loss: 0.49496 - acc: 0.8373 -- iter: 16160/20311
Training Step: 20826  | total loss: 0.48396 | time: 122.947s
| Momentum | epoch: 033 | loss: 0.48396 - acc: 0.8411 -- iter: 16192/20311
Training Step: 20827  | total loss: 0.46726 | time: 123.197s
| Momentum | epoch: 033 | loss: 0.46726 - acc: 0.8476 -- iter: 16224/20311
Training Step: 20828  | total loss: 0.47357 | time: 123.467s

**Comment by willduan** [comment]
I think you can see that through tensorboard

**Comment by willduan** [comment]
Oh, maybe you can see this example:
[https://github.com/tflearn/tflearn/blob/master/examples/images/dnn.py](https://github.com/tflearn/tflearn/blob/master/examples/images/dnn.py)

```
# Regression using SGD with learning rate decay and Top-3 accuracy
sgd = tflearn.SGD(learning_rate=0.1, lr_decay=0.96, decay_step=1000)
top_k = tflearn.metrics.Top_k(3)
net = tflearn.regression(softmax, optimizer=sgd, metric=top_k,
                         loss='categorical_crossentropy')
```

**Comment by Kevin-Moon** [comment]
@willduan Thank you for the exact answer! :)
